linters/pre-commit-hooks/plugin.yaml

```diff
@@ -0,0 +1,20 @@
+version: 0.1
+lint:
+  definitions:
+    - name: pre-commit-hooks
+      known_good_version: 4.4.0
+      runtime: python
+      package: pre-commit-hooks
+      environment:
+        - name: PATH
+          list: ["${env.PATH}"]
+      commands:
+        - name: end-of-file-fixer
+          enabled: false
```

Comment:
Please also add `suggest_if: never` to the linter definition for it.

See `sqlfluff` `preCheck` in the test for getting this green